src/ManagerClass.js

```diff
@@ -2,8 +2,15 @@ import Customer from '../src/CustomerClass.js'
 class Manager extends Customer {
   constructor() {
     super();
+    this.users;
   }
 
+  customerSearch(search) {
+    let users = this.users.filter(user => {
+      return user.name.toLowerCase().includes(search)
+    })
+    return users
+  }
```

Comment:
Is there there functionality somewhere to also make sure the search value is lowercase?

Reply:
Oh good  call, I will add that.